lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/AccessRequirementUtils.java

```diff
@@ -211,21 +222,25 @@ public static void validateAccessRequirementAcl(AccessControlList acl) {
 			ValidateArgument.required(access.getPrincipalId(), "acl.resourceAccess.principalId");
 			ValidateArgument.requirement(!BOOTSTRAP_PRINCIPAL.ANONYMOUS_USER.getPrincipalId().equals(access.getPrincipalId()), "Cannot assign permissions to the anonmous user.");
 			ValidateArgument.requirement(!BOOTSTRAP_PRINCIPAL.PUBLIC_GROUP.getPrincipalId().equals(access.getPrincipalId()), "Cannot assign permissions to the public group.");
-			Set<ACCESS_TYPE> accessSet = access.getAccessType();
-			ValidateArgument.requirement(accessSet.stream()
-							.allMatch(access_type -> access_type == ACCESS_TYPE.REVIEW_SUBMISSIONS || access_type == ACCESS_TYPE.EXEMPTION_ELIGIBLE),
-					"Only the REVIEW_SUBMISSION and EXEMPTION_ELIGIBLE ACCESS_TYPE are supported for access requirements.");
 		});
 	}
 
+	/**
+	 * @param acl
+	 * @param objectType
+	 * This method checks if acl has allowed access type for specific object type.
+	 * It throws IllegalArgumentException if acl contains unauthorized access type.
+	 */
 	public static void validateResourceAccessOfAclForOwnerType(AccessControlList acl, ObjectType objectType) {
```

Comment:
Since this method used by all types that use ACL it should not be in a class called:  AccessRequirementUtils.
Maybe move it to something like AclUtils